src/export_balances.ts

```diff
@@ -0,0 +1,67 @@
+import create_api from './api'
+import { ApiPromise } from '@polkadot/api'
+import { Profile, MemberId } from '@joystream/types/members'
+import { Option } from '@polkadot/types/'
+import { AccountId } from '@polkadot/types/interfaces'
+
+main()
+
+async function main () {
+    const api = await create_api()
+    // WIP
+    console.log(await enumerate_member_accounts(api))
+    api.disconnect()
+}
+
```

Comment:
This is still a Work in progress will update in later PR today hopefully.

Reply:
basic implementation done in 0b787aa7d6da4196351d4e3a1177a265ef0f75bb
will improve in future PR